internal/rbd/nodeserver.go

```diff
@@ -145,11 +145,8 @@ func (ns *NodeServer) NodeStageVolume(
 	}
 	defer cr.DeleteCredentials()
 
-	if acquired := ns.VolumeLocks.TryAcquire(volID); !acquired {
-		util.ErrorLog(ctx, util.VolumeOperationAlreadyExistsFmt, volID)
-		return nil, status.Errorf(codes.Aborted, util.VolumeOperationAlreadyExistsFmt, volID)
-	}
-	defer ns.VolumeLocks.Release(volID)
+	// considering kubelet make sure node operations like publish/stage...etc can not be called
```

Comment:
@humblec are we sure that kubelet is taking some central locking when it comes to RWO on multiple nodes. I past we have seen that RWO request comes up on two nodes parallel and if this happens we end up corrupting the data?

Reply:
Discard this one. I was confused with watchers